features/step_definitions/basic_steps.rb

```diff
@@ -5,3 +5,12 @@
 Then(/^I should see "([^"]*)"$/) do |content|
   expect(page).to have_content content
 end
+
+Given(/^I click "([^"]*)"$/) do |arg1|
+  click_link_or_button arg1
+end
```

Comment:
Sorry guys, but change `arg1` to a variable name that makes sense: `button` or `link`